src/k8s-engine/kim.ts

```diff
@@ -125,20 +129,42 @@ class Kim extends EventEmitter {
     return this._isReady;
   }
 
-  async runCommand(args: string[], sendNotifications = true): Promise<childResultType> {
-    const child = spawn(resources.executable('kim'), args);
+  async runKimCommand(args: string[], sendNotifications = true): Promise<childResultType> {
+    return await this.runCommand(spawn(resources.executable('kim'), args), args[0], sendNotifications);
+  }
+
+  async runTrivyCommand(args: string[], sendNotifications = true): Promise<childResultType> {
+    let child: ChildProcess;
+    const kimCommand = args[0];
+
+    if (os.platform().startsWith('win')) {
+      args = ['-d', 'rancher-desktop', 'trivy'].concat(args);
+      child = spawn('wsl', args);
+    } else if (os.platform().startsWith('darwin')) {
+      const limaBackend = this.k8sManager as LimaBackend;
+
+      args = ['trivy'].concat(args);
+      child = limaBackend.limaSpawn(args);
```

Comment:
I'll file a follow up to add some kind of shell execution to the `KubernetesBackend` interface.

Reply:
That was #445